# Generated by Django 3.1 on 2020-09-20 08:35

import apps.users.validators
import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('users', '0016_auto_20200913_1317'),
    ]

    operations = [
        migrations.AlterField(
            model_name='user',
            name='username',
            field=models.CharField(db_index=True, error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. From 6 to 30 characters. Letters, digits and @/./+/-/_ only.', max_length=30, unique=True, validators=[apps.users.validators.UnicodeUsernameValidator(), django.core.validators.MinLengthValidator(6), django.core.validators.MaxLengthValidator(30)], verbose_name='username'),
        ),
    ]
